lisa/tools/lspci.py

```diff
@@ -142,47 +165,107 @@ def _install(self) -> bool:
         return self._check_exists()
 
     def get_device_names_by_type(
-        self, device_type: str, force_run: bool = False
+        self, device_type: str, force_run: bool = False, use_pci_ids: bool = False
```

Comment:
It looks the new argument is to filter devices by name? If so, you can add a new method called `get_devices_by_allowed_list`, it calls get_device_names_by_type and filter them.

Reply:
Removed usage of a parameter "use_pci_ids" from all places. Its no longer needed.